src/integrationTest/java/uk/gov/hmcts/darts/arm/service/ArmResponseFilesProcessorIntTest.java

```diff
@@ -927,21 +971,30 @@ void givenProcessResponseFilesSuccessfullyCompletesForAnnotation() throws IOExce
         when(armDataManagementConfiguration.getTempBlobWorkspace()).thenReturn(fileLocation);
 
         String uploadFileTestFilename = "tests/arm/service/ArmResponseFilesProcessorTest/uploadFile/" +
-                "6a374f19a9ce7dc9cc480ea8d4eca0fb_04e6bc3b-952a-79b6-8362-13259aae1895_1_uf.rsp";
+            "6a374f19a9ce7dc9cc480ea8d4eca0fb_04e6bc3b-952a-79b6-8362-13259aae1895_1_uf.rsp";
         String uploadFileJson = TestUtils.getContentsFromFile(uploadFileTestFilename);
         BinaryData uploadFileBinaryData = BinaryData.fromString(uploadFileJson);
         when(armDataManagementApi.getBlobData(uploadFileFilename)).thenReturn(uploadFileBinaryData);
 
-        doNothing().when(armDataManagementApi).deleteResponseBlob(inputUploadBlobFilename);
```

Comment:
not sure why we call this one BlobFileName but the others are just FileName - since they are all just files in the blob store. Probably not too important in a test!

Reply:
Yep, it wasn't something I touched, but agree to rename it from `BlobFilename` to `Filename` throughout this test class